views/books-show.html

```diff
@@ -0,0 +1,27 @@
+<div class="container">
+  <hr>
+    <div class="clearfix">
```

Comment:
might be able to stop shuffle behavior by starting a new bootstrap row instead of clearfix